compiler/rustc_resolve/src/lib.rs

```diff
@@ -534,8 +534,8 @@ struct BindingKey {
     /// identifier.
     ident: Ident,
     ns: Namespace,
-    /// 0 if ident is not `_`, otherwise a value that's unique to the specific
-    /// `_` in the expanded AST that introduced this binding.
+    /// 0 if ident is not `_` or ``, otherwise a value that's unique to the specific
+    /// `_`/`` in the expanded AST that introduced this binding.
```

Comment:
https://github.com/rust-lang/rust/pull/115367 was reverted in https://github.com/rust-lang/rust/pull/131045, and even if it weren't `kw::Empty` shouldn't have been used for this anyway.

Reply:
Huh, interesting. `empty_disambiguator` was added in #115367 but wasn't removed by the revert in #131045. @compiler-errors, you did the revert, do you remember how you went about it -- was it a straightforward reversion? Was leaving `empty_disambiguator` behind an oversight?

There is one test (`tests/ui/dollar-crate/dollar-crate-is-keyword.rs`) that hits the empty identifier path. But if I remove `empty_disambiguator` all tests pass. So I've removed the commit changing this comment and added a new commit that removes `empty_disambiguator`.